src/mpi/coll/ibcast/ibcast_tsp_ring_algos.h

```diff
@@ -22,8 +22,7 @@
 #undef FCNAME
 #define FCNAME MPL_QUOTE(FUNCNAME)
 int MPIR_TSP_Ibcast_sched_intra_ring(void *buffer, int count, MPI_Datatype datatype, int root,
-                                     int tag, MPIR_Comm * comm, int maxbytes,
-                                     MPIR_TSP_sched_t * sched)
+                                     MPIR_Comm * comm, int maxbytes, MPIR_TSP_sched_t * sched)
```

Comment:
Unrelated whitespace?

Reply:
No, this is a consequence of removing `tag` argument and then the style-checker asking to collate the lines.